skimage/exposure/exposure.py

```diff
@@ -825,7 +891,7 @@ def is_low_contrast(image, fraction_threshold=0.05, lower_percentile=1,
         return not ((image.max() == 1) and (image.min() == 0))
 
     if image.ndim == 3:
-        from ..color import rgb2gray, rgba2rgb  # avoid circular import
+        from ..color.colorconv import rgb2gray, rgba2rgb  # no circular import
```

Comment:
I tested it locally, it seems that this circular import disappeared :tada: Can you confirm @grlee77?

Reply:
Did you test with EAGER_IMPORT=1 defined? (It failed on CI with a circula import on a previous run, so I  added it back in)

Reply:
> Did you test with EAGER_IMPORT=1 defined? (It failed on CI with a circula import on a previous run, so I added it back in)

Nop, I just ran `pytest` locally :sweat_smile:... Too bad :confused: